jscomp/syntax/ast_attributes.ml

```diff
@@ -97,13 +97,15 @@ let process_attributes_rev (attrs : t) : attr_kind * t =
       | "bs", (Nothing | Uncurry _)
         ->
         Uncurry attr, acc (* TODO: warn unused/duplicated attribute *)
+#if BS_NATIVE = false then
       | "bs.this", (Nothing | Meth_callback _)
         ->  Meth_callback attr, acc
       | "bs.meth",  (Nothing | Method _)
         -> Method attr, acc
       | "bs", _
       | "bs.this", _
         -> Bs_syntaxerr.err loc Conflict_bs_bs_this_bs_meth
+#end
```

Comment:
would it do any harm if we keep it

Reply:
Good point, this doesn't need to be there.